packages/playground-examples/copy/ja/3-7/Types and Code Flow/Assertion Functions.ts

```diff
@@ -0,0 +1,65 @@
+//// { compiler: {  }, order: 1 }
+
+// JavaScriptの柔軟性を考えると、ランタイムチェックにコードを追加して
+// 仮定を検証するというのは良い考えかもしれません。
+
+// この検証を行うコードは一般的にアサーション(または不変条件)とよばれ、
+// 変数が期待と一致しない場合に早期にエラーを発生させる
+// 小さな関数です。
+
+// Nodeには標準でこれを行うための、assertと呼ばれる関数が付属しており、
+// インポートなしで使用することができます。
+
+// でもここでは、自前で定義してみましょう。 valueという式が
```

Comment:
IMO: でもだと少し口語っぽい感じがするので
```suggestion
// しかし、ここでは自前で定義してみましょう。 valueという式が
```